Client/TabManagement/Legacy/LegacySavedTab+ConfigureExtension.swift

```diff
@@ -48,9 +48,13 @@ extension LegacySavedTab {
         if let screenshotUUID = screenshotUUID, let imageStore = imageStore {
             tab.screenshotUUID = screenshotUUID
             if let uuidString = tab.screenshotUUID?.uuidString {
-                imageStore.get(uuidString) >>== { screenshot in
-                    if tab.screenshotUUID == screenshotUUID {
-                        tab.setScreenshot(screenshot)
+                Task {
+                    do {
+                        let screenshot = try await imageStore.getImageForKey(uuidString)
+                        if tab.screenshotUUID == screenshotUUID {
+                            tab.setScreenshot(screenshot)
+                        }
+                    } catch {
                     }
```

Comment:
If you aren't going to do anything in the catch block then you can do try? instead of a do catch block. Same thing for the rest of the do's in the PR.

Reply:
I wasn't sure if I should handle the errors somehow, like either logging them or showing the user some kind of notice.